Doc/tutorial/introduction.rst

```diff
@@ -100,10 +100,8 @@ give you an error::
 There is full support for floating point; operators with mixed type operands
 convert the integer operand to floating point::
 
-   >>> 3 * 3.75 / 1.5
-   7.5
-   >>> 7.0 / 2
-   3.5
+   >>> 3 * 3.75 - 1
```

Comment:
Wouldn't be better to use the example that takes the integer result? There is nothing unexpected that `3 * 3.75` results a `float`, it can't be an `int`. But `4 * 3.75` is an integer of type `float` even if it can be represented as `int`.